reduction/lr_reduction/instrument_settings.py

```diff
@@ -0,0 +1,28 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class InstrumentSettings:
+    """
+    Dataclass to store the instrument geometry parameters.
+    Default values determined from settings.json
+
+    Attributes
+    ----------
+    source_detector_distance : float - Distance from the source to the detector in meters
+    sample_detector_distance : float - Distance from the sample to the detector in meters
+    num_x_pixels : int - Number of pixels in the x direction
+    num_y_pixels : int - Number of pixels in the y direction
+    pixel_width : float - Width of the pixels in mm
+    xi_reference : float - xi reference value in mm
+    s1_sample_distance : float - Distance from the sample to the s1 aperture in mm
+    """
+
+    apply_instrument_settings: bool = False
```

Comment:
I'm confused about the intent of this field. As far as I understant, the instrument settings  _always_ apply, be it from the infamous `settings.json` file and/or later modified by the user.